crates/cli-tools/src/changelog.rs

```diff
@@ -49,13 +49,51 @@ impl Display for Severity {
 
 #[derive(Debug, Default, PartialEq, Eq)]
 struct Changelog {
+    crate_path: String,
```

Comment:
We should start using `PathBuf` and `impl AsRef<Path>` like `crates/cli/src/main.rs` and `crates/cli-tools/src/actions.rs`. This can be done in a separate PR though.

Reply:
Will do in separate PR